src/ecmult_compute_table.h

```diff
@@ -0,0 +1,16 @@
+/*****************************************************************************************************
+ * Copyright (c) 2013, 2014, 2017, 2021 Pieter Wuille, Andrew Poelstra, Jonas Nick, Russell O'Connor *
+ * Distributed under the MIT software license, see the accompanying                                  *
+ * file COPYING or https://www.opensource.org/licenses/mit-license.php.                              *
+ *****************************************************************************************************/
+
+#ifndef SECP256K1_ECMULT_COMPUTE_TABLE_H
+#define SECP256K1_ECMULT_COMPUTE_TABLE_H
+
+/* Construct table of all odd multiples of gen in range 1..(2**(window_g-1)-1). */
```

Comment:
`window_g-2`?

Reply:
`ECMULT_TABLE_SIZE(w)` is `(1L << ((w)-2))`. But that's just the size of the table, not the range. As the entries are two apart (only odd values), I think the range extends to 2^(w-1)-1.